specs/protocol/predeploys.md

```diff
@@ -71,6 +72,7 @@ or `Bedrock` or `Canyon`. Deprecated contracts should not be used.
 | SchemaRegistry                | 0x4200000000000000000000000000000000000020 | Bedrock    | No         | Yes     |
 | EAS                           | 0x4200000000000000000000000000000000000021 | Bedrock    | No         | Yes     |
 | BeaconBlockRoot               | 0x000F3df6D732807Ef1319fB7B8bB8522d0Beac02 | Ecotone    | No         | No      |
+| OperatorFeeVault              | 0x420000000000000000000000000000000000001B | Bedrock    | No         | Yes     |
```

Comment:
```suggestion
| OperatorFeeVault              | 0x420000000000000000000000000000000000001B | Isthmus    | No         | Yes     |
```
Operator Fee Vault was introduced as part of Isthmus hardfork

ref https://github.com/alloy-rs/hardforks/blob/542524e879f600a6b7fa80cd49ca40fb6716ad67/crates/op-hardforks/src/optimism/mainnet.rs#L21-L22
ref https://specs.optimism.io/protocol/isthmus/overview.html